packages/apps/docs/src/components/Layout/components/Footer/styles.css.ts

```diff
@@ -53,15 +48,15 @@ export const linkClass = style([
   sprinkles({
     display: 'block',
     textDecoration: 'none',
-    color: '$neutral6',
+    color: '$gray40',
     paddingX: '$3',
     textAlign: 'center',
   }),
   {
     selectors: {
       '&:hover': {
-        textDecoration: 'underline',
```

Comment:
remvoing the underline here is fine. there is a high contrast with color and background color